src/services/NurseryBatchService.ts

```diff
@@ -27,28 +27,28 @@ const BATCH_HISTORY_ENDPOINT = '/api/v1/nursery/batches/{batchId}/history';
 export const BATCH_PHOTO_ENDPOINT = '/api/v1/nursery/batches/{batchId}/photos/{photoId}';
 
 const DEFAULT_BATCH_FIELDS = [
-  'id',
+  'accession_id',
+  'accession_accessionNumber',
+  'addedDate',
   'batchNumber',
+  'facility_id',
+  'facility_name',
   'germinatingQuantity',
+  'germinatingQuantity(raw)',
+  'id',
+  'notes',
   'notReadyQuantity',
+  'notReadyQuantity(raw)',
+  'project_id',
+  'readyByDate',
   'readyQuantity',
+  'readyQuantity(raw)',
+  'subLocations.subLocation_id',
+  'subLocations.subLocation_name',
   'totalQuantity',
   'totalQuantityWithdrawn',
-  'germinatingQuantity(raw)',
-  'notReadyQuantity(raw)',
-  'readyQuantity(raw)',
```

Comment:
Changes are due to putting them in alphabetical order, the net here is that `project_id` was added